str/str_test.go

```diff
@@ -0,0 +1,68 @@
+package str
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestAfter(t *testing.T) {
+
+	assert.Equal(t, "nah", After("hannah", "han"))
```

Comment:
I start each test with empty values as parameters. As an example:
`assert.Equal(t, "", After("", ""))`
`assert.Equal(t, "", After("hannah", ""))`
`assert.Equal(t, "", After("", "hannah"))`
(I'm not sure if the result is what it should be.)

Would you like to add that to this test and check it in the other tests?

Reply:
Good idea.